slide.sh

```diff
@@ -26,13 +35,24 @@ function slide() {
             printf -vBARE "%${COLS}s" '' && BARE=${BARE// /-}
             LINE=${LINE//\!\!sep/$BARE}
         fi
-        [ ${#MESSAGE} -lt $COLS ] && MSGPOS=$(((COLS-1)-${#MESSAGE}))
         [ ${#BARE} -le $COLS ] && CTRPOS=$(((COLS-${#BARE})/2))
         [ $CENTER -eq 1 ] && $TPUT cup $LINENUM $CTRPOS || $TPUT cup $LINENUM 0
         printf -- "${LINE//%/%%}\n"
+        sleep .1
```

Comment:
Again I don't think this forced delay really adds anything, other than the perception that slides are loading from some remote resource like a database or URL.